### ISSUE TITLE
WCAG fail of Drawer

### ISSUE THREAD

**Issue by robmausser** [adaptlearning/adapt-contrib-core#103]
Collapsed state of hamburger menu (Drawer) are not announced. Provide Collapsed for hamburger menu.

https://www.screencast.com/t/ar1EXj15R

**Comment by oliverfoster** [comment]
Just to note: `aria-expanded` is normally used to label a toggle button that is focusable when the content is both expanded and collapsed. With the drawer, the toggle button is not focusable when the drawer is open, such that the expanded state will never be read.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded
https://www.w3.org/WAI/GL/wiki/Using_the_WAI-ARIA_aria-expanded_state_to_mark_expandable_and_collapsible_regions
http://accessibility.athena-ict.com/aria/examples/menu.shtml

Examples are all non-modal dialogs, lists or trees.
